doc/usecases/gamification/reward_preview.md

```diff
@@ -0,0 +1,27 @@
+# Use Case: Reward Preview
+
+## Summary
+
+- **Scope:** Reward Preview
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** A user can see a preview of future rewards for his efforts.
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  - The user has opened the app and is editing a map.
+  - The user has planted at least one plant on his map.
+- **Main success scenario:**
+  - The app calculates the dates for receiving potential rewards of the existing plants, like e.g. the flowering time or harvesting periods of a certain plant.
+  - The user can see a marker on the [map timeline](map_timeline.md) indicating those rewards.
+  - When selecting a date or timeframe that includes such a reward, the icon of the respective plant(s) will change to include a bed of flowers for plants with 
+    active flowering time and a harvest bucket for plants that yield harvest at that moment.
```

Comment:
This is very hard to read. Please make sentences shorter and easier to read. (It was unclear for us what exactly you mean.)

Maybe this is not a use case by itself and just an extension for the smoothie use case of @Benjaminimal?

Reply:
Which use case are you referring to, so I can maybe include my thoughts there.

Reply:
Iirc you were asked for review there. Please do the requested reviews first, e.g. #167

Reply:
I went through the list of use cases I was requested for review but I found no mention of a smoothie use case. The only thing similar I can think of is my own use case in #162.